config/locales/en.yml

```
@@ -2,7 +2,15 @@ en:
   title: MovieTicketBooking
   english: "EN"
   vietnam: "VN"
-  sign_up: "Sign up"
+  sessions:
+    new:
+      title: "Login"
+      name_btn: "Log in"
+      new_user: "New user? %{link_sigup}"
```

nên dùng i18n html để tránh phải truyền link_to và dùng html_safe
```suggestion
      new_user_html: 'New user? <a href="%{link_sigup}"> ...</a>'
```